extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1282,6 +1282,7 @@ export class AmpA4A extends AMP.BaseElement {
           'fallback to 3p disabled');
       return Promise.resolve(false);
     }
+    // TODO(keithwrightbos): remove when no longer needed.
```

Comment:
The linter might complain about the absence of an issue number here, but if it doesn't care, I don't care.